app/views/shared/_lessons_table.slim

```diff
@@ -7,6 +7,8 @@ table class='mdl-data-table mdl-js-data-table resource-table'
       th class='mdl-data-table__cell--non-numeric' Group Name
       == render partial: 'shared/ordered_column', locals: { order_key: :date, column_name: 'Date' }
       th class='mdl-data-table__cell--non-numeric' Subject
+      th class='mdl-data-table__cell--non-numeric' Total Students
+      th class='mdl-data-table__cell--non-numeric' Graded Students
```

Comment:
Let's remove `class='mdl-data-table__cell--non-numeric'` no both of these since they are numeric values. We use those only for well non-numeric values :)